pymultitor.py

```diff
@@ -316,9 +317,9 @@ def load(loader):
         )
         loader.add_option(
             name="on_status_code",
-            typespec=int,
-            default=0,
-            help="change ip when a specific status code returned",
+            typespec=Sequence[str],
```

Comment:
There's no `Sequence[int]` in mitmproxy:

- https://github.com/mitmproxy/mitmproxy/blob/1f2e1a254a68e54e60360b66762ed6bdf2ff26be/docs/src/content/addons-commands.md?plain=1#L120
- https://github.com/mitmproxy/mitmproxy/blob/1f2e1a254a68e54e60360b66762ed6bdf2ff26be/mitmproxy/optmanager.py#L368